parts/kubernetesagentcustomdata.yml

```diff
@@ -14,7 +14,7 @@ write_files:
   content: |
     [Service]
     ExecStart=
-    ExecStart=/usr/bin/docker daemon -H fd:// --storage-driver=overlay
+    ExecStart=/usr/bin/docker daemon -H fd:// --storage-driver=overlay --log-driver=journald --bip={{WrapAsVariable "dockerBridgeCidr"}}
```

Comment:
we purposefully reverted the changes to use journald as it broke lots of monitoring tech. Could you go back and make sure you are properly merged with that change